doc/client_api.md

```diff
@@ -331,6 +331,26 @@ localStream.updateSimulcastActiveLayers({0: true, 1: false});
 
 In this example we are disabling layer 1 while the other layer (0) is active.
 
+
+## Add handlers to the mediaStream
```

Comment:
Part of this documentation is outdated. Also, I'd rather move it to `server_api.md`.

Reply:
Changed